finat/physically_mapped.py

```diff
@@ -179,3 +200,21 @@ def physical_edge_lengths(self):
            edge (numbered according to FIAT conventions), shape
            (nfacet, ).
         """
+
+    @abstractmethod
+    def physical_points(self, point_set, entity=None):
+        """Maps reference element points to GEM for the physical coordinates
+
+        :arg point_set: A point_set on the reference cell to push forward to physical space.
+        :arg entity: Reference cell entity on which the point set is
+                     defined (for example if it is a point set on a facet).
+        :returns: a GEM expression for the physical locations of the
+                  points, shape (gdim, ) with free indices of the point_set.
+        """
+
+    @abstractmethod
+    def physical_vertices(self):
+        """Physical locations of the cell vertices.
+
+        :returns: a GEM expression for the physical vertices, shape
+                (gdim, )."""
```

Comment:
This could be implemented in terms of `physical_points`.  However, this non-minimality does not look too bad.